conversation/src/main/java/com/ibm/watson/developer_cloud/conversation/v1/ConversationService.java

```diff
@@ -940,6 +958,9 @@ public ConversationService(String versionDate, String username, String password)
     if (createValueOptions.synonyms() != null) {
       contentJson.add("synonyms", GsonSingleton.getGson().toJsonTree(createValueOptions.synonyms()));
     }
+    if (createValueOptions.patterns() != null) {
```

Comment:
I'm not sure the `patterns` support is finalized.  Might want to leave that out.